src/FINDOLOGIC/Export/Data/Item.php

```diff
@@ -396,6 +393,17 @@ public function addProperty(Property $property): void
     }
 
     /**
+     * Adds an attribute. If attributes with the same key, but a different value are added, they will not
```

Comment:
`...they will be merged` would be more clear than `...they will not be overridden`, because the latter requires knowledge of the behavior before the change, which will not necessarily be there in the future.